docs/root/mithril/mithril-protocol/protocol.md

```diff
@@ -3,126 +3,119 @@ sidebar_position: 1
-* _When using bulletproof_: Generate a "random" string, for example by taking some unpredictable data from the net (hash
-  of latest block, stockmarket quotes) and hash it
-* When requested by a party, return the parameters and random string
+* Choose/establish a prime order group to serve as the foundation for the multi-signature scheme
+* _When implementing bulletproof_: generate a pseudo-random string, such as hashing unpredictable data from the net (eg, latest block's hash, stock market quotes)
+* Upon request from a party, provide the parameters and a random string.
 
-This setup phase must be done only once and is valid for as long as the same proof system, e.g the parameters can very well be statically baked into the various systems producing and consuming Mithril proofs.
+This setup phase must be done only once and is valid for as long as the same proof system (eg, the parameters might be statically baked into the various systems producing and consuming Mithril proofs).
 
 Similarly, during the protocol establishment phase, the parameters `k`, `m` and `phi_f` are defined, as well as the set of allowed signers.
 
-### 2. Initialisation Phase
+### 2. Initialisation phase
 
-This phase includes both the actual initialisation phase of each party, and the registration of keys.
+This phase includes both the actual initialization phase of each party and the registration of keys.
 
-* Party retrieves the protocol parameters
-* It uses the parameters to generate a new key pair of a verification key and a signing key, plus a Proof-of-possession
-  `κ`. The verification key and PoP are broadcast to all other parties for registration, which is expected to last for
-  some limited period of time. An important difference with the paper is that registration is not centralised in the "
-  idealised" manner described in the paper. Instead, all signers will register against all other signers. Meaning that
-  each signer will trace it's own registration procedure. Together with other participants' keys, the stake of each
-  party is stored.
-* Then an aggregate verification key (`AVK` in the paper) is created from the registration material, in the form of a
-  Merkle-tree.
+Firstly, the party retrieves the protocol parameters.  
 
-The key dissemination process can also happen on-chain, which makes sense as the parties need to have some guarantees about the validity of keys and stakes of each other party.
+Next, they use the parameters to generate a new key pair of a verification key and a signing key, plus a proof of possession (PoP)
+  `κ`. The verification key and PoP are broadcast to all other parties for registration, which is expected to last for a limited period. An important difference between the implementation and the paper is that registration is not centralized in the 'idealized' manner described in the paper. Instead, all signers will register against all other signers. Thus
```

Comment:
Here it says that registration is not centralised. Maybe we want to change that? @jpraynaud

Reply:
It looks OK, as it says that the registration is ideally not centralized, but is centralized in the current implementation IIUC. WDYT @iquerejeta?

Reply:
Oh, you are right. I misread. It says that a difference is that in the paper registration is not centralised. Sorry, my bad 🙏  no further comments.

Reply:
No problem, you made me doubt :sweat_smile: